lib/item.js

```diff
@@ -10,7 +10,7 @@ export const defaultCommentSort = (pinned, bio, createdAt) => {
   return 'hot'
 }
 
-export const isJob = item => typeof item.maxBid !== 'undefined'
+export const isJob = item => item.maxBid !== null && typeof item.maxBid !== 'undefined'
```

Comment:
This has been preventing the image proxy from proxying `item.url` since maybe forever.

Reply:
`typeof null === 'object'`

oof

I think with polymorphic items this wouldn't have happened since we'd have something like `item.type`?

Reply:
yep!